OasisDEX/oasis-borrow
# Add navigation links to header.

# [Add navigation header to not connected navigation bar for Borrow](https://app.clubhouse.io/oazo-apps/story/2347/add-navigation-header-to-not-connected-navigation-bar-for-borrow)

<please insert a clubhouse link above>
  
![Screen Shot 2021-09-01 at 14 03 10](https://user-images.githubusercontent.com/653586/131713345-ab1628dd-9b76-4725-b94a-4e464a56524b.png)

## Changes 👷‍♀️
  <Please add short list of changes>

- Add navigation header to top that shows up when the app is disconnected.
- Mobile menu.
- Add Github icon at the footer.
  
## How to test 🧪
  <Please explain how to test your changes>
- Run locally and check that all works properly.
    
## Definition of done ✔️
